api/envoy/type/matcher/value.proto

```diff
@@ -0,0 +1,58 @@
+syntax = "proto3";
+
+package envoy.type.matcher;
+option go_package = "matcher";
+
+import "envoy/type/matcher/number.proto";
+import "envoy/type/matcher/string.proto";
+
+import "validate/validate.proto";
+
+// [#protodoc-title: ValueMatcher]
+
+// Specifies the way to match a ProtobufWkt::Value. Primitive values and ListValue are supported.
+// StructValue is not supported and is always not matched.
+message ValueMatcher {
```

Comment:
Although the diff looks a bit scary, IIUC this refactoring out `Value` -> `ValueMatcher` shouldn't break backward compatibility of those protos, right?

Reply:
Yeah from my eye LGTM, but would like if @htuch could double check.